### DISCUSSION TITLE
How can I customize the syntax color scheme for "Variables"?

### DISCUSSION THREAD

**Discussion by Devansh80** [BeardedBear/bearded-theme#75] [Q&A] [answered]
### I want to change the syntax color for "variables or constant" and "object".  I don't like this pink color for variables-

![3](https://user-images.githubusercontent.com/46435891/162901557-c97c5168-23a2-4e0a-aea6-8934dff962a4.png)

### Instead i want to make it white or maybe little grey. At the time of vscode launch theme looks exactly same as i want but after some seconds the theme suddenly changed to pink like below-

![1](https://user-images.githubusercontent.com/46435891/162902271-c931f876-b114-48fc-be38-66cf24554d49.png)
![2](https://user-images.githubusercontent.com/46435891/162902299-c8bb5860-a8e1-4a20-9622-1516aa650cf4.png)

Please help me

**Comment by BeardedBear** [comment, marked as answer]
You have 2 ways to do this : 
1. Adding a overcharging rules for specific theme in your `settings` file for VSCode (but you lost semantic coloring), for example : 
```json
"editor.tokenColorCustomizations": {
    "[Bearded Theme Monokai Terra]": {
      "variables": {
        "foreground": "#FF0000"
      }
    }
  }
```

2. Fork the repository, modify the variable colors, build it locally, and import customized `.vsix` file in extension panel

**Reply by Devansh80** [reply]
Thank you!

**Reply by Devansh80** [reply]
Hey could you help me to build it locally ?

**Reply by BeardedBear** [reply]
You can follow the Readme file here for installation / building : https://github.com/BeardedBear/bearded-theme/blob/master/README-DEV.md

But in short, after forking repository : 
1. `yarn install` or `npm install` for installing dependencies
2. Modify colors 
   - for scopes in [semantic file](https://github.com/BeardedBear/bearded-theme/blob/master/src/scopes/semanticTokens.ts) or [scopes file](https://github.com/BeardedBear/bearded-theme/blob/master/src/scopes/scopes.ts)
   - for a [specific theme](https://github.com/BeardedBear/bearded-theme/tree/master/src/variations)
3. `yarn start` or `npm run start` for launch dev mode, then `F5` for opening an instance for see modifications
4. `yarn build:ext` or `npm run build:ext` for build extension
5. import the generated local file `.vsix` in extension panel

**Reply by Devansh80** [reply]
Hey thanks for your help. It is working perfectly as I want.
But here I had to add an extra step to make it work. I also changed the json data according to the color i needed.

**Reply by Devansh80** [reply]
If I want to change the keyword font style is there any way?
Like if i want to make every single keyword bold or maybe italic then can i do that also??

**Reply by BeardedBear** [reply]
check the doc ;) https://code.visualstudio.com/docs/getstarted/themes
